assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-selection-panel.scss

```
@@ -60,11 +60,24 @@
 
 			.googlesitekit-audience-selection-panel__error-notice-actions {
 
-				.googlesitekit-cta-link {
-					color: $c-utility-on-error-container;
-					font-size: $fs-label-md;
-					font-weight: $fw-medium;
-					line-height: $lh-label-md;
+				.googlesitekit-audience-selection-panel__error-notice-action {
```

Please take a look at my above comments.

```suggestion
				.mdc-button--tertiary {
```

---

Responded above, thanks Hussain!